live-examples/css-examples/fonts/font-size-adjust.css

```diff
@@ -0,0 +1,24 @@
+@font-face {
+    font-family: 'Fira Sans';
+    src: local('FiraSans-Regular'),
+         url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+    font-weight: normal;
+    font-style: normal;
+}
+
+#output section {
+    flex-direction: column;
+}
+
+.unadjusted {
+    font-family: serif;
+    font-size: 20px;
+}
+
+.unadjusted code {
+    font-size-adjust: .45;
+}
+
+#example-element {
+    font-family: 'Fira Sans', sans-serif;
```

Comment:
I'm not sure why this example uses different `font-family` and `font-size` values for adjusted and unadjusted. I think it would be better to keep everything the same except the `font-size-adjust` value.

Or am I missing something?

Reply:
OK, there's a few issues here:

- The `font-size` was a mistake on my part (the example text was all supposed to have the same `font-size` and I goofed up something fierce with git stash).
- The different `font-family` properties were meant to illustrate the intended use (as described in the spec) for dealing with `font-size` discrepancies caused by different `font-family` values.
- I'll try it again with one `font-family` only, but I wonder if it would confuse why you want to use this property—that it's some alternate to `font-size` or something like that.